### PR TITLE
Remove unused HTTP request data in call to /api/lock_status

### PR DESCRIPTION
**Pull request by maddiedierker**
Follow-up to #29746.

I noticed that the `teacherPanelHelpers#queryLockStatus` method was sending unused request data to the server, so just cleaning that up while I'm here.

The client was sending `user_id` and `script_id` in its request, which [`/api/lock_status`](https://github.com/code-dot-org/code-dot-org/blob/f0d7655268f2d46af91359847e40cc7df7524c42/dashboard/app/controllers/api_controller.rb#L145-L167) does not use.

### CODE REVIEW

File: apps/src/code-studio/teacherPanelHelpers.js [hunk 1]
```diff
@@ -61,14 +60,9 @@ export function renderTeacherPanel(
  * Query the server for lock status of this teacher's students
  * @returns {Promise} when finished
  */
-export function queryLockStatus(store, scriptId) {
+export function queryLockStatus(store) {
   return new Promise((resolve, reject) => {
-    $.ajax('/api/lock_status', {
-      data: {
-        user_id: clientState.queryParams('user_id'),
-        script_id: scriptId
```
Comment (davidsbailey) [inline apps/src/code-studio/teacherPanelHelpers.js:69]: hmmmm it looks like script_id might be accessed via this call stack:
https://github.com/code-dot-org/code-dot-org/blob/3ec25ef0b213fdb121492ef9b4e6452ffd94542e/dashboard/app/controllers/api_controller.rb#L146
https://github.com/code-dot-org/code-dot-org/blob/3ec25ef0b213fdb121492ef9b4e6452ffd94542e/dashboard/app/controllers/api_controller.rb#L154
https://github.com/code-dot-org/code-dot-org/blob/3ec25ef0b213fdb121492ef9b4e6452ffd94542e/dashboard/app/controllers/api_controller.rb#L472-L473

user_id looks safe to remove, though.

**Comment by maddiedierker** [comment]
@davidsbailey this is a tiny clean-up PR, but PTAL!

**Review by davidsbailey — COMMENTED** [review]
ohh, sorry! wrote some comments but never published 😳

File: apps/src/code-studio/teacherPanelHelpers.js (hunk 1 above)
Comment (maddiedierker) [reply apps/src/code-studio/teacherPanelHelpers.js:69]: 😱 good find!

**Review by davidsbailey — APPROVED** [review]

**Pull request merged**